### PR TITLE
Fixes and features

### LINKED ISSUE #189: Require is not defined
After downloading the deb package and trying to see if this would actually give me desktop notifications I get this error:

image
image

### PR DESCRIPTION
**Pull request by MagiusCHE**
### Done
- Better client injector method focused to separate js and css files to maintenance (suggeted by @joax but i think mine is a better solution in our case).
- Integrated some features from @joax 
  - JS and CSS Injected into child windows 8c91e9b5be83805c8c18f587a474df2d1f0a0da5
  - added unreadMessage observer handler mentioned by @joax as alternative handler (see next)
- Added multiple handler for unreadMessage observer to handle custom urls be7307b31a742dcaa8f9c2c660430c85d9ccd947
- Fix #189, #135
- New feature: context menu cut/paste/etc... #173
- New feature: context menu Show link url, Copy link url, Copy link label. #122

### CODE REVIEW

File: src/controller/mail-window-controller.js [hunk 1]
```diff
@@ -52,8 +52,8 @@ class MailWindowController {
                 spellcheck: true,
                 nativeWindowOpen: true,
                 affinity: 'main-window',
-/*                contextIsolation: false,
-                nodeIntegration: true,*/
+                contextIsolation: false,
+                nodeIntegration: true,
             }
         })
```
[the remark this answers is not in the document]
Comment (MagiusCHE) [reply src/controller/mail-window-controller.js:58]: ingore

[the remark this answers is not in the document]
Comment (MagiusCHE) [reply src/controller/mail-window-controller.js:58]: @sonatype-lift help

**Comment by MagiusCHE** [comment]
We should ignore lift warning if we accept and need to use `require` at client side.

File: src/controller/mail-window-controller.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MagiusCHE) [reply src/controller/mail-window-controller.js:58]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (MagiusCHE) [reply src/controller/mail-window-controller.js:58]: @sonatype-lift ignore

**Comment by cavemandaveman** [comment]
Any chance of this getting merged?

**Review by julian-alarcon — APPROVED** [review]
Hi! I appreciate your contributions! BTW it will be easier to review small PR for the next time.

**Pull request merged**